Sources/ApolloCodegenLib/Templates/RenderingHelpers/String+SwiftNameEscaping.swift

```diff
@@ -130,7 +130,7 @@ enum SwiftKeywords {
     "try",
   ]
 
-  fileprivate static let InputParameterNamesToEscape: Set<String> = FieldAccessorNamesToEscape
+  static let InputParameterNamesToEscape: Set<String> = FieldAccessorNamesToEscape
```

Comment:
Does the access level on this need to change? It doesn't appear to be used anywhere new in this PR.

Reply:
Fixed